### DISCUSSION TITLE
Bind key only during completion

### DISCUSSION THREAD

**Discussion by unrealapex** [magicant/yash#72] [Q&A] [answered]
I would like to bind <kbd>h</kbd>, <kbd>j</kbd>, <kbd>k</kbd>, and  <kbd>l</kbd> to navigate the completion menu:
```sh
bindkey --emacs 'h' complete-prev-column
bindkey --emacs 'j' complete-next-candidate
bindkey --emacs 'k' complete-prev-candidate
bindkey --emacs 'l' complete-next-column

```
but I am not able to do that without being prevented from using the keys normally when the completion menu is closed. Would it be possible to make these keys behave normally if the completion menu isn't open?

**Comment by unrealapex** [comment]
@magicant is this possible or is this a feature request?

**Reply by magicant** [reply, marked as answer]
This is not possible now. Something like #61 would be needed.
